tools/test/test262/excludelist.orig.xml

```diff
@@ -4427,9 +4428,9 @@
     <test id="built-ins/Proxy/create-handler-is-revoked-proxy"><reason>TODO</reason></test>
     <test id="built-ins/Proxy/create-target-is-revoked-function-proxy"><reason>TODO</reason></test>
     <test id="built-ins/Proxy/create-target-is-revoked-proxy"><reason>TODO</reason></test>
-    <test id="built-ins/Proxy/defineProperty/targetdesc-not-configurable-writable-desc-not-writable"><reason>TODO</reason></test>
-    <test id="built-ins/Proxy/deleteProperty/targetdesc-is-configurable-target-is-not-extensible"><reason>TODO</reason></test>
-    <test id="built-ins/Proxy/getOwnPropertyDescriptor/resultdesc-is-not-configurable-not-writable-targetdesc-is-writable"><reason>TODO</reason></test>
+    <test id="built-ins/Proxy/defineProperty/return-boolean-and-define-target"><reason>TODO</reason></test>
+    <test id="built-ins/Proxy/set/trap-is-missing-receiver-multiple-calls"><reason>TODO</reason></test>
+    <test id="built-ins/Proxy/set/trap-is-missing-receiver-multiple-calls-index"><reason>TODO</reason></test>
```

Comment:
You have excluded some TCs (even some test262 cases) for this patch. 
Is there any conflict between old and new standard?
Would you please explain this?